package-lock.json

```diff
@@ -10,6 +10,7 @@
             "dependencies": {
                 "@duckduckgo/autofill": "github:duckduckgo/duckduckgo-autofill#4.7.0",
                 "@duckduckgo/content-scope-scripts": "github:duckduckgo/content-scope-scripts#2.1.1",
+                "@duckduckgo/ddg2dnr": "github:duckduckgo/ddg2dnr#0.1.2",
```

Comment:
This code needs a review also. Is there a good place to do this?

Reply:
Good catch, I had just assumed (yes that word) that had been PR'd.

Reply:
Yea, I suppose this is a fair time to review the ddg2dnr code. Are you volunteering @jonathanKingston?

I'm not sure the best way to do that initial review, if you like you could go through the code and email me feedback. I could then prepare a PR with your requested changes, and assign it back to you for a review?

Reply:
BTW, do you feel strongly about blocking this PR on the review of ddg2dnr? Ideally I'd like to land the changes in this review now/soon, then make requested ddg2dnr changes as a follow up, then finally update the ddg2dnr dependency again to include those changes. Just because that would save me some pain keeping on rebasing these changes.

Reply:
_(For the record, we discussed this in Asana and agreed to do a follow-up review of ddg2dnr.)_